EventFilter/CTPPSRawToDigi/plugins/CTPPSPixelDigiToRaw.h

```diff
@@ -0,0 +1,55 @@
+#ifndef CTPPSPixelDigiToRaw_H
```

Comment:
.h file is not really needed for producers and the content can be merged to the corresponding .cc file